.github/workflows/prune-self-hosted-runners.yml

```
@@ -2,6 +2,8 @@ name: Prune Self-Hosted Runners
 
 on:
   workflow_dispatch:
+  schedule:
+    - cron: '0 * * * *' # Hourly
```

```suggestion
    - cron: '27 2 * * *' # Daily, @ ~2:27AM
```

A little more random-ish.